src/node/internal/inspect_polyfill.ts

```diff
@@ -0,0 +1,3924 @@
+// @ts-nocheck
+
+// `util.inspect` polyfill
+//
+// Copied built version from https://www.npmjs.com/package/node-inspect-extracted/v/2.0.2?activeTab=code, then converted to IIFE from UMD, and formatted with prettier.
```

Comment:
We need to be sure to include the license copyright information from the polyfill as a comment block in here.